packages/snaps-rpc-methods/src/restricted/getBip32Entropy.test.ts

```diff
@@ -232,44 +299,52 @@ describe('getBip32EntropyImplementation', () => {
         }
       `);
 
-      expect(getMnemonic).toHaveBeenCalledWith('source-id');
+      expect(getMnemonicSeed).toHaveBeenCalledWith('source-id');
```

Comment:
```suggestion
      expect(getMnemonicSeed).toHaveBeenCalledWith('source-id');
      expect(getMnemonic).not.toHaveBeenCalled();
```

In other test files too.

Reply:
Added!